### PR TITLE
refactored the page Bootstrap and tables

### PR DESCRIPTION
**Pull request by Ashaghel**
using tables will help with exporting to SQL tables later on if wanted to make a dynamic app 
Bootstrap because CSS is too hard
CYF logo because why not

### CODE REVIEW

**Comment by Augs0** [comment]
@supertata why dont you be the first reviewer for Ahmad's suggested changes? They are quite big changes! :D Go to the 'files changed' tab and click the review changes button. Then you can choose whether to approve or reject and send him some comments~ Good luck and ask any questions if you're unsure. Feel free to choose two other people to review this change too!

File: index.html [hunk 1]
```diff
@@ -1,16 +1,79 @@
...
+            alt="CYF"
+            class="card-img logo"
+          />
+        </div>
+        <div class="col-md-8">
+          <div class="card-body">
+            <h5 class="card-title">CYF dictionary</h5>
+            <p class="card-text">
+              This a collaborative dictionary made for and by the team of CYF
+              students and mentors to help everyone new to Software Developmern.
```
Comment (supertata) [inline index.html]: @Ashaghel there is a little typo in the word "Development"

Comment (Ashaghel) [reply index.html]: fixed

**Comment by supertata** [comment]
Hi @zoltan-gal @mhallaq I'm adding you two to review these changes, to keep in line with the rule.

**Pull request merged**